macros/stitch/base/stitch_bing_keyword_performance.sql

```diff
@@ -13,6 +12,85 @@ with source as (
     
 ),
 
+renamed as (
+
+    select
+    
+        "__SDC_PRIMARY_KEY" as keyword_performance_report_id,
+
+        convert_timezone('UTC', timeperiod)::timestamptz::date 
+            as campaign_date,
+            
+        accountid as account_id,
+        adgroupid as ad_group_id,
+        adgroupname as ad_group_name,
+        adid as ad_id,
+        campaignid as campaign_id,
+        campaignname as campaign_name,
+        keywordid as keyword_id,
+        
+        campaignstatus as campaign_status,
+        
+        coalesce(finalurl, '') as url,
+        
+        clicks,
+        impressions,
+        spend,
+        
+        rank() over (
+            partition by timeperiod::date 
+            order by _sdc_report_datetime desc
+            ) as rank
+
+    from source
+
+),
+
+parsed as (
+
+    select
+    
+        keyword_performance_report_id,
+        campaign_date,
+        keyword_id,
+        account_id,
+        ad_group_id,
+        ad_group_name,
+        ad_id,
+        campaign_id,
+        campaign_name,
+        campaign_status,
+        
+        url,
+        split_part(split_part(split_part(url, '//', 2),'/', 1),'?',1)::varchar as url_host,
+        '/' || split_part(url, '/', 4)::varchar as url_path,
+        split_part(split_part(url,'utm_source=',2), '&', 1) as utm_source,
```

Comment:
switch these to be `dbt_utils.get_url_parameter()`

Reply:
done (long over-due, lol)